src/patterns/service-unavailable-pages/index.md.njk

```diff
@@ -0,0 +1,104 @@
+---
+title: Service unavailable pages
+description: This is a page that tells someone a service is unavailable. It should say when the service will be available or what to do if it is permanently closed.
+section: Patterns
+theme: Pages
+aliases:
+backlog_issue_id: 124
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This pattern is currently experimental because <a class="govuk-link" href="#research-on-this-pattern">more research</a> is needed to validate it.
+---
+{% from "_example.njk" import example %}
+
+This is a page that tells someone a service is unavailable. It should say when the service will be available or what to do if it is permanently closed. These are also known as 503 and shutter pages.
+
+{{ example({group: 'patterns', item: 'service-unavailable-pages', example: 'default', html: true, open: false}) }}
+
+## When to use this pattern
+
+Use a service unavailable page when a service has been closed on purpose. This could be for a specific period of time or permanently.
+
+If there is a problem with the service, use a technical problem page.
```

Comment:
Change to "If there is a problem with the service, use a there is a problem with the service page." This should link to those types of pages, which have been renamed.

Reply:
Since they both dont exist yet, we'll have to add these links after they're both live...